tests/upgrade/test_crossgrade.sh

```diff
@@ -43,8 +43,13 @@ usage() {
 }
 
 ISTIO_NAMESPACE="istio-system"
-# Minimum % of all requests that must be 200 for test to pass.
-MIN_200_PCT_FOR_PASS="85"
+# Maximum % of 503 response that cannot exceed
+MAX_503_PCT_FOR_PASS="15"
+# Maximum % of connection refused that cannot exceed
+# Set it to high value so it fails for explicit sidecar issues
```

Comment:
if there are explicit sidecar issues, could you expand on those?  There shouldn't be any that I am aware of...

Reply:
I think ideally there would be some connectivity issue only during upgrade, so it should be only small percentage. So if there is high percentage of that appears, it implies that there might be actual data plane issues.

Reply:
For a better understanding - why is 15% 503 more stable than 85% 200? What were the other requests? Is this expected?

15% is quite high, isn't it?

Reply:
The main purpose for this change is to separate check of two different groups of errors to make the result more clear. Previously we checked only the 200 code percentage so we may not know whether there is actual issue during upgrade or it is just intermittent failure caused by small jump of connection errors.